node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/maintenance/acl/AclMaintainer.java

```diff
@@ -43,10 +48,19 @@
 
     private final ContainerOperations containerOperations;
     private final IPAddresses ipAddresses;
+    private final Metrics metrics;
+    private final Map<String, Long> lastSuccess;
+    private static final String METRIC_NAME_POSTFIX = ".acl.age";
 
-    public AclMaintainer(ContainerOperations containerOperations, IPAddresses ipAddresses) {
+    public AclMaintainer(ContainerOperations containerOperations, IPAddresses ipAddresses, Metrics metrics) {
         this.containerOperations = containerOperations;
         this.ipAddresses = ipAddresses;
+        this.metrics = metrics;
+        this.lastSuccess = new HashMap<>(){{
```

Comment:
This creates an anonymous subclass of `HashMap`, which is not ideal. Consider doing `new HashMap<>(Map.of(...))` instead.

Consider also extracting a variable for `System.currentTimeMillis` so that both versions have the same initial timestamp.